src/UI/Component/Symbol/Icon/Standard.php

```diff
@@ -127,17 +132,18 @@ interface Standard extends Icon
     const CMIS = 'cmis';	//xAPI/cmi5
     const REP = 'rep';	//Repository
     const TASK = 'task';   //Task
+    const LS = 'lso';   //Learning Sequence
```

Comment:
```suggestion
    const LSO = 'lso';   //Learning Sequence
```
Let's try to stick with "LSO" =)